config/vufind/SierraRest.ini

```diff
@@ -268,3 +268,15 @@ title_hold_excluded_item_codes = "e"
 ;display_orders = false
 ; Whether to check holdings records in search results (default is true):
 ;check_holdings_in_results = true
+
+; This section can be configured to allow display of held and checked out items from
+; INN-Reach systems
+[InnReach]
+; Toggle enabled to true to turn on InnReach Integration
+enabled = false
+; In order to get title information from INN-Reach we need to connect to Sierra DNA
+;This requires the PostgreSQL extension to be enabled in PHP
+sierra_db = postgresql://[DNA URL]:[DNA Port]/[DNA Database]?user=[DNA User]&password=[DNA password]&sslmode=prefer
```

Comment:
This line seems to be causing a parse error and breaking some tests. You should probably put the value inside double quotes and/or comment it out by default.